tsconfig.json

```diff
@@ -9,8 +9,7 @@
     "strict": true,
     "skipLibCheck": true,
     "outDir": "dist",
-    "sourceMap": true,
-    "declaration": true
```

Comment:
Why don't we need declarations anymore?

Reply:
We still need it. But `declaration: true` is already defined in line 5 of the tsconfig.json (therefore this one was a duplicate)